dask_ml/linear_model/utils.py

```diff
@@ -1,33 +1,73 @@
 """
 """
+import dask
 import dask.array as da
 import dask.dataframe as dd
 import numpy as np
 from multipledispatch import dispatch
 
+try:
+    import dask_expr
 
-@dispatch(dd._Frame)
-def exp(A):
-    return da.exp(A)
+    _dask_expr_avail = True
+except ImportError:
+    dask_expr = None
+    _dask_expr_avail = False
 
 
-@dispatch(dd._Frame)
-def absolute(A):
-    return da.absolute(A)
+if dask.config.get("dataframe.query-planning") and _dask_expr_avail:
```

Comment:
I think this is incorrect, starting from Friday we will enable query planning if the value is None, so it's probably better if you use a utility function that does this check

Reply:
nvm, I thought maybe this was better: https://github.com/dask/dask-ml/pull/980/commits/864e5adee128e2575538bd45f70b9db4770c372a